utils/generate_types/overrides.d.ts

```diff
@@ -222,6 +222,19 @@ type AccessibilityNode = {
 export const selectors: Selectors;
 export const devices: Devices & DeviceDescriptor[];
 
+export type SecurityDetails = {
+  issuer?: string,
+  protocol?: string,
+  subjectName?: string,
+  validFrom?: number,
+  validTo?: number,
+};
+
+export type RemoteAddr = {
+  ipAddress: string,
+  port: number,
+};
+
```

Comment:
These types kept not ending up in `types.d.ts`, so I added them here. Is that expected or should they have automatically showed up in `types.d.ts`. I was running…

```
$ node utils/generate_types/
$ node utils/generate_channels.js
$ npm run build
$ npm run lint 
```